src/pyfmi/master.pyx

```diff
@@ -756,7 +756,7 @@ cdef class Master:
             index_end = index_start + self.models_dict[model]["local_output_len"]
             local_output_vref_array = (<FMUModelCS2>model).get_real(self.models_dict[model]["local_output_vref_array"])
             for i, index in enumerate(range(index_start, index_end)):
-                y[index] = local_output_vref_array[i]
+                y[index] = local_output_vref_array[i].item()
```

Comment:
`model.get_real `will always return a numpy array, setting a scalar with an array is deprecated.

+ a few more occasions in this file.